ios/FluentUI/People Picker/AvatarViewVnext.swift

```diff
@@ -0,0 +1,340 @@
+public enum AvatarVnextPresence: Int, CaseIterable {
+    case none
+    case available
+    case away
+    case busy
+    case doNotDisturb
+    case outOfOffice
+    case offline
+    case unknown
+    case blocked
+
+    public func color() -> UIColor? {
+        var color: UIColor?
+
+        switch self {
+        case .none:
+            break
+        case .available:
+            color = Colors.Palette.presenceAvailable.color
+        case .away:
+            color = Colors.Palette.presenceAway.color
+        case .busy:
+            color = Colors.Palette.presenceBusy.color
+        case .doNotDisturb:
+            color = Colors.Palette.presenceDnd.color
+        case .outOfOffice:
+            color = Colors.Palette.presenceOof.color
+        case .offline:
+            color = Colors.Palette.presenceOffline.color
+        case .unknown:
+            color = Colors.Palette.presenceUnknown.color
+        case .blocked:
+            color = Colors.Palette.presenceBlocked.color
+        }
+
+        return color
+    }
+
+    public func image() -> UIImage? {
+        var imageName: String?
+
+        switch self {
+        case .none:
+            break
+        case .available:
+            imageName = "ic_fluent_presence_available_16_filled"
+        case .away:
+            imageName = "ic_fluent_presence_away_16_filled"
+        case .busy:
+            imageName = "ic_fluent_presence_busy_16_filled"
+        case .doNotDisturb:
+            imageName = "ic_fluent_presence_dnd_16_filled"
+        case .outOfOffice:
+            imageName = "ic_fluent_presence_oof_16_regular"
+        case .offline:
+            imageName = "ic_fluent_presence_offline_16_regular"
+        case .unknown:
+            imageName = "ic_fluent_presence_unknown_16_regular"
+        case .blocked:
+            imageName = "ic_fluent_presence_blocked_16_regular"
+        }
+
+        var image: UIImage?
+        if let imageName = imageName {
+            image = UIImage.staticImageNamed(imageName)
+        }
+
+        return image
+    }
+}
+
+@objc(MSFAvatarVnextState)
+/// Properties available to customize the state of the avatar
+public class AvatarVnextState: NSObject, ObservableObject {
+    @objc @Published public var image: UIImage?
+    @objc @Published public var primaryText: String?
+    @objc @Published public var secondaryText: String?
+    @objc @Published public var ringColor: UIColor?
+    @objc @Published public var backgroundColor: UIColor?
+    @objc @Published public var foregroundColor: UIColor?
+    @objc @Published public var presence: AvatarVnextPresence = .none
+}
+
+/// Representation of design tokens to buttons at runtime which interfaces with the Design Token System auto-generated code.
+/// Updating these properties causes the SwiftUI button to update its view automatically.
+public class AvatarTokens: ObservableObject {
+    @Published public var avatarSize: CGFloat!
+    @Published public var borderRadius: CGFloat!
```

Comment:
borderradius seems weird naming?

Reply:
I think that's the naming design uses in the spec?

Reply:
Correct, @chiuam! This is the naming being used on the figma files the design team for the Vnext specs.

Reply:
I've added a comment to follow up on that and ensure the naming convention being used is intentional and consistent across all platforms.